modules/utils/monitor_phy_param.py

```diff
@@ -0,0 +1,52 @@
+import getopt
+import sys
+import datetime
+import subprocess
+from time import sleep
+from threading import Thread
+
+#defult rssi monitoring interval: 5sec
+rssi_mon_interval = 5
```

Comment:
Only asking, how should we determine the most optimal interval time?

Reply:
This we need to derive based on few experiments with diff radio.
If we have motion awareness we can have aggressive interval in comparison to the static scenario.
This will be mostly overridden by the cmdline param.